hooks/useLockout.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+
+/**
+ * Function to constantly check the current time every hour and set the lockout accordingly.
+ * @returns - The value of lockedOut.
+ */
+const useLockout = (): boolean => {
```

Comment:
Since this hook returns a binary value to indicate a user's ability to perform a task, would a more specific name be effective?

Ex: `useIsUserLockedOut`

Reply:
I actually like this name. I moreso just used useLockout to match everything. I'll change it :)

Reply:
@braydoncoyer this has been updated by @alexappleget